packages/core/content-type-builder/server/src/services/builder.ts

```diff
@@ -13,20 +13,34 @@ export const getReservedNames = () => {
     ],
     // attributes are compared with snake_case(name), so only snake_case is needed here and camelCase + UPPER_CASE matches will still be caught
     attributes: [
-      // TODO V5: these need to come from a centralized place so we don't break things accidentally in the future
+      // TODO: these need to come from a centralized place so we don't break things accidentally in the future and can share them outside the CTB, for example on Strapi bootstrap prior to schema db sync
+
+      // ID fields
       'id',
       'documentId',
       'document_id',
+
+      // Creator fields
       'created_at',
       'updated_at',
       'published_at',
       'created_by_id',
       'updated_by_id',
-      'locale', // conflicts with internal locale field
-
-      // not actually breaking but we'll leave it to avoid confusion
+      // does not actually conflict because the fields are called *_by_id but we'll leave it to avoid confusion
       'created_by',
       'updated_by',
+
+      // Used for Strapi functionality
+      'entry_id',
+      'strapi',
+      'status',
+      'localizations',
+      'meta',
+      'locale',
+      'stage',
+      'release',
+      'history',
+      'version',
```

Comment:
are we sure we need these 4? they look like quite common words to use

For stage we use `strapi_stage` (and `strapi_assignee`)
And I believe release, history and version will not need to extend the content types (cc @Feranchz @remidej )